1_fetch.R

```diff
@@ -118,6 +131,41 @@ p1_targets_list <- list(
   tar_target(
     p1_reaches_sf,
     st_read(p1_reaches_shp)
+  ),
+
+  # fetch prms met data
+  tar_target(
+    p1_prms_met_data_zip_file,
+    download_sb_file(sb_id = "5f6a289982ce38aaa2449135",
+                     file_name = "sntemp_inputs_outputs_drb.zip",
+                     out_dir = "1_fetch/out"),
+    format = "file"
+  ),
+
+  # unzip prms met data
+  tar_target(
+    p1_prms_met_data_csv_file,
+    {
+    unzip(zipfile=p1_prms_met_data_zip_file,exdir = dirname(p1_prms_met_data_zip_file),overwrite=TRUE)
+    file.path(dirname(p1_prms_met_data_zip_file), "sntemp_inputs_outputs_drb.csv")
+    },
+    format = "file"
+  ),
+
+  # read in prms met data
+  tar_target(
+    p1_prms_met_data,
+    read_csv(p1_prms_met_data_csv_file)
```

Comment:
Can add `show_col_types = FALSE` as an argument to `read_csv` to suppress messages about interpreted column data types with pipeline build.